### PR TITLE
Pr4 5 sjofn

### PR DESCRIPTION
**Pull request by tzatushevskaya**
please approve without checking

### CODE REVIEW

File: public/Sjofn/practice/sjofn4-5.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    function isEven(a) {
+        return a % 2 === 0;
+
+    }
+
+    let yourGeneratedNumber = generateRandomInt();
+    console.log(`${yourGeneratedNumber} is ${isEven(yourGeneratedNumber) ? 'Even' : 'Odd'}`);
+
+    function generateRandomIntInRange(min, max) {
+        return Math.floor(Math.random() * (max - min + 1)) + min;
```
Comment (eclimov) [inline public/Sjofn/practice/sjofn4-5.html]: According to the task, the function should also print some text in the console. Please, check.

**Review by SandaPanteleimonov — APPROVED** [review]
Fine, approved, let's go.

**Review by SandaPanteleimonov — APPROVED** [review]
Approved, let's go.

File: public/Sjofn/practice/sjofn4-5.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
     function generateRandomInt() {
         let randomNum = Math.floor(Math.random()*10);
-        console.log(`Random integer: ${randomNum}`);
+        console.log(`${randomNum} is ${isEven(randomNum) ? 'Even' : 'Odd'}`);
```
Comment (eclimov) [inline public/Sjofn/practice/sjofn4-5.html]: You need to print different text here, according to the task. 
Please, fix.

**Review by eclimov — APPROVED** [review]
Perfect!
Approved.

**Pull request merged**